BrickKit/build.gradle

```diff
@@ -4,25 +4,34 @@ buildscript {
     }
 
     repositories {
-        jcenter()
         google()
         gradlePluginPortal()
+        mavenCentral()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:4.1.2'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.21"
```

Comment:
why not use Kotlin 1.4.32 which is the latest?

Reply:
Let's update it next time!